src/selectors/edges.js

```diff
@@ -48,10 +66,17 @@ export const getTransitiveEdges = createSelector(
           return;
         }
         const target = edgeTargets[edgeID];
+
+        // Further filter out connections between indicative input / output nodes under focus mode
+        const isNotInputEdge =
```

Comment:
It is a performance thing as checking `isNotInputEdge` is more straight forward and has less cases rather than the positive situation ( this is apparent if you read through the condition - checking for isInputEdge has much more possibility rather than checking for `isNotInputEdge`. ) 

Out of curiosity, I wonder what is the upside of the refactor work for flipping this to positive?